.github/workflows/GAP.yml

```diff
@@ -0,0 +1,32 @@
+name: "GAP tests"
+
+# Trigger the workflow on push or pull request
+on:
+  - push
+  - pull_request
+
+jobs:
+  test:
+    name: "GAP ${{ matrix.gap-branch }} on ${{ matrix.os }}"
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        digraphs-lib:
+          - digraphs-lib-0.6
+        gap-branch:
+          - master
+          - stable-4.11
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: gap-actions/setup-gap-for-packages@master
+        with:
+          GAP_PKGS_TO_CLONE: "wilfwilson/NautyTracesInterface"
+          GAP_PKGS_TO_BUILD: "io orb profiling grape NautyTracesInterface datastructures"
+          GAPBRANCH: ${{ matrix.gap-branch }}
+      - name: "Install digraphs-lib"
+        run: curl --retry 5 -L -O https://digraphs.github.io/Digraphs/${{ matrix.digraphs-lib }}.tar.gz
+             && tar xf ${{ matrix.digraphs-lib }}.tar.gz
+      - uses: wilfwilson/run-test-for-packages@master
```

Comment:
Same, what does this do? Where's the source code?

Reply:
Where you see lines like this in a GitHub workflow, this is referring to a GitHub repository that hosts a GitHub workflow.

It is saying: at this point, run the current `master` branch of the workflow `run-test-for-packages` of the user/organisation `wilfwilson`. So the code that it is running is the workflow here: https://github.com/wilfwilson/run-test-for-packages

This is a fork of the workflow https://github.com/gap-actions/run-test-for-packages

I will use the `gap-actions` workflow directly, once the PR that I have made to it (https://github.com/gap-actions/run-test-for-packages/pull/1) has been merged.

The purpose of this step is to is to load GAP and run the Digraphs tests, and upload code coverage data. Ultimately, it runs the script https://github.com/gap-system/pkg-ci-scripts/blob/master/run_tests.sh which loads GAP and checks that the file `tst/testall.g` that I introduce below returns a 0 exit code.

Reply:
This now uses the `gap-actions` workflow directly, the reference to my `wilfwilson` fork is gone.